docs/integration-guides/key-management.md

```diff
@@ -454,17 +454,24 @@ curl -d '{
 }
 ```
 
-!!! info
-    Below are a few helpful pieces of information to consider:
+!!! tip "Block watching and re-work"
+    Since V20.0, blocks processed using [`process`](/commands/rpc-protocol/#process) are placed under observation by the node. If you wish to disable this feature, add `"watch_work": "false"` to the process RPC command.
```

Comment:
@guilhermelawless Maybe add "...are placed under observation by the node for possible re-generation of work under certain conditions."

Reply:
Thanks, also added re-broadcasting.